### DISCUSSION TITLE
Looking for a nextjs starter-kit to replace my current meteorjs stack

### DISCUSSION THREAD

**Discussion by hk86** [vercel/next.js#12853] [Help]
Hello there,

I'm currently investigating whether I should switch from meteorjs to nextjs. Currently I use these packages and features:

- meteorjs
- redux for state management
- mongodb as my database
- react-intl to support multiple languages, each language has it's own path i.e. domain.com/fr
- I make use of meteors live update
- semantic ui as my css framework

I'm looking to switch to:

- nextjs
- apollo/graphql
- I want to keep mongodb
- tailwindcss
- the site should be available for multiple languages

My questions:

- Is there some nextjs starter-kit available, which fits my needs? 
- How do I achieve live updates on nextjs?
- How do I build a multi language site with paths for each language?
- I guess there is no need for redux if I use apollo/graphql?

Best regards,

HK

**Comment by leerob** [comment]
Hey 👋 You're going to love Next.js. 

There isn't a starter that contains everything _together_ but there are plenty of examples for each individually. 

- [Apollo](https://github.com/zeit/next.js/tree/canary/examples/with-apollo)
- [Tailwind CSS](https://github.com/zeit/next.js/tree/canary/examples/with-tailwindcss)
- [i18n](https://github.com/isaachinman/next-i18next/tree/master/examples/simple)
- MongoDB would be agnostic to whatever front-end you choose

> How do I achieve live updates?

Live updates (aka "hot module reloading") are included out of the box.

> I guess there is no need for Redux if I use Apollo/GraphQL?

There is probably no need for Redux using Apollo's cache. If you do truly need global state, I would recommend [React Context](https://reactjs.org/docs/context.html).
